sys-meta/sh/sh-0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="/bin/sh (POSIX shell) symlink"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+SRC_URI=""
+S=${WORKDIR}
+
+LICENSE="metapackage"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="+bash dash lksh mksh"
+REQUIRED_USE="^^ ( bash dash lksh mksh )"
+
+RDEPEND="
+	bash? ( app-shells/bash )
+	dash? ( app-shells/dash )
+	lksh? ( app-shells/mksh[lksh] )
+	mksh? ( app-shells/mksh )
```

Comment:
`app-shells/ksh` is the at&t `ksh93` + enterprise distro patches from redhat/suse/solaris/aix and so on. it's the real deal and provides posix mode if needed.

```
Invocation.
...
       If the basename is sh or rsh, or the -o posix option is given on the invocation command line, then the  shell  is
       initialized  in full POSIX compliance mode
```

Reply:
I suppose you're asking me to add ksh.